manifests/a/Axosoft/GitKraken/9.2.1/Axosoft.GitKraken.installer.yaml

```diff
@@ -0,0 +1,28 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Axosoft.GitKraken
+PackageVersion: 9.2.1
+InstallerLocale: en-US
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+Scope: user
+InstallModes:
+- interactive
+- silent
+InstallerSwitches:
+  Silent: /silent
+  SilentWithProgress: /silent
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://release.axocdn.com/win64/GitKrakenSetup-9.2.1.exe
+  InstallerSha256: 157789AA51B1087BDD6A74231CF21E01F4E455729F3837D74F2FEF0E5DE94067
+- Architecture: x86
+  InstallerType: exe
+  InstallerUrl: https://release.axocdn.com/win32/GitKrakenSetup-9.2.1.exe
+  InstallerSha256: DAB184393D1C734C03F64BA0BF6B85C99B10A39C2545EDC5C440497A3CC654D1

```

Comment:
SHA256 Mismatch: 
Correct Sha256 is B0536BFDF0808701667F5B22CA088F4583E4BD86FA2CF0AC6950638DBDC73FC5